PIL/Image.py

```diff
@@ -28,8 +28,11 @@
 
 from PIL import VERSION, PILLOW_VERSION, _plugins
 
+import logging
 import warnings
 
+logger = logging.getLogger(__name__)
```

Comment:
Filters and handlers are already able to control output of children loggers, so I don't see how this matters.